Sources/StreamChatUI/CommonViews/ContainerStackView.swift

```diff
@@ -32,6 +32,32 @@ extension ContainerStackView {
 
         private let rawValue: Int
     }
+
+    /// Describes the Spacing between the arranged subviews.
+    public struct Spacing: Equatable, ExpressibleByFloatLiteral, ExpressibleByIntegerLiteral {
+        /// The actual value of the Spacing.
+        public var rawValue: CGFloat
+
+        /// `Spacing` can be expressed by a `Double`.
+        /// Example: `spacing = 10.0`, instead of `spacing = Spacing(10.0)`.
+        public init(floatLiteral value: Double) {
+            rawValue = CGFloat(value)
+        }
+
+        /// `Spacing` can be expressed by an `Int`.
+        /// Example: `spacing = 10`, instead of `spacing = Spacing(10)`.
+        public init(integerLiteral value: Int) {
+            rawValue = CGFloat(value)
+        }
+
+        init(_ rawValue: CGFloat) {
+            self.rawValue = rawValue
+        }
```

Comment:
I believe this should be `public`, too. If you can do `spacing = 5`, you should be able to do `spacing = Spacing(myConstant)`, like you do in tests :)

Reply:
Yes nice catch 👍 Thanks!